src/lib.rs

```diff
@@ -417,11 +445,24 @@ impl MenuEvent {
         }
     }
 
-    pub(crate) fn send(event: MenuEvent) {
+    pub fn send(event: MenuEvent) {
         if let Some(handler) = MENU_EVENT_HANDLER.get_or_init(|| None) {
             handler(event);
         } else {
             let _ = MENU_CHANNEL.0.send(event);
         }
     }
 }
+
+#[cfg(feature = "ksni")]
```

Comment:
these should also be cfg'd behind linux-only target and same for all related logic

Reply:
it should result in faster compile times on other platforms, no?

Reply:
You're right. I didn't notice that the ksni feature is enabled on all platforms.
I added the target_os cfg everywhere, except `platform_impl/gtk`.